### PR TITLE
feat(files): add support for last modified timestamp

### PR DESCRIPTION
**Pull request by josephmcg**
**What this PR does** 📖
- store modified in Item class
- dynamically set modified for a Directory based on latest modified value of a child
- update modified on like/share
- refresh every minute on list view

**Which issue(s) this PR fixes** 🔨
AP-907

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by stavares843** [comment]
the /rebase will not work due to the conflict

**Comment by josephmcg** [comment]
never tried it before lol I was wondering what the issue was

**Comment by stavares843** [comment]
works as long there's no conflict 😂 and sorry about the cypress conflict

File: components/views/files/list/List.vue [hunk 1]
```diff
@@ -30,8 +30,18 @@ export default Vue.extend({
...
+    /**
+     * counter to force reactivity for Map
+     */
+    counter: {
+      type: Number,
+      required: true,
+    },
+  },
+  created() {
+    setInterval(this.forceRender, this.$Config.chat.timestampUpdateInterval)
```
Comment (molimauro) [inline components/views/files/list/List.vue:44]: Can we save the interval and then clear it upon component destruction?

**Review by Jekrimo — APPROVED** [review]

File: components/views/files/list/List.vue (hunk 1 above)
Comment (josephmcg) [reply components/views/files/list/List.vue:44]: Sure, could you explain the benefits of doing so? I don't understand

**Comment by stavares843** [comment]
is possible to have invalid date here? @josephmcg 

i haven't saw any invalid date but i was wondering if we have code to prevent that

also, does this appear with the specific date when is more than 24h?

minor questions, merging

**Review by stavares843 — APPROVED** [review]

**Pull request merged**